### PR TITLE
Copyedits to Cookies reference

### PR DESCRIPTION
**Pull request by wizardofdocs**

### CODE REVIEW

File: site/en/docs/extensions/reference/cookies/index.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ to access. For example:
...
-origin of the top level frame. This means that if site A is embedded using an iframe in site B
-and site C, a partitioned cookie can have a different value in each.
+origin of the top-level frame. This means that, for example, if site A is embedded using an iframe in site B
+and site C, a partitioned cookie can have a different value on each site. <!--does that mean all 3, or just B and C?-->
 
-`chrome.cookies` does not support partitioning, which means that all methods
+Because `chrome.cookies` doesn't support partitioning, all of its methods
 read and write cookies from all partitions. The [`cookies.set()`](#method-set) method stores cookies in
-the default partition.
+the default partition. <!--So cookies can be partitioned, but not using the cookies API. Am I understanding that right?-->
```
Comment (jpmedley) [inline site/en/docs/extensions/reference/cookies/index.md:33]: cc @oliverdunk 

That's probably right. Partitioning is very new and I haven't heard of any enhancements to `cookies.set()`.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/reference/cookies/index.md (hunk 1 above)
Comment (oliverdunk) [reply site/en/docs/extensions/reference/cookies/index.md:33]: Right. And Chrome decides how they are partitioned.

File: site/en/docs/extensions/reference/cookies/index.md [hunk 2]
```diff
@@ -25,12 +25,12 @@ to access. For example:
 ## Partitioning {: #partitioning }
 
 [Partitioned cookies][chips] allow a site to mark that certain cookies should be keyed against the
-origin of the top level frame. This means that if site A is embedded using an iframe in site B
-and site C, a partitioned cookie can have a different value in each.
+origin of the top-level frame. This means that, for example, if site A is embedded using an iframe in site B
+and site C, a partitioned cookie can have a different value on each site. <!--does that mean all 3, or just B and C?-->
```
[the remark this answers is not in the document]
Comment (oliverdunk) [reply site/en/docs/extensions/reference/cookies/index.md:29]: A single cookie name `foo` can have three different values (or more accurately, can exist three times in different partitions) in each of the following cases:

- When it's the top level frame you navigated to
- When it's embedded on site A
- When it's embedded on site B

[the remark this answers is not in the document]
Comment (wizardofdocs) [reply site/en/docs/extensions/reference/cookies/index.md:29]: so can I say "a different value on each of the three sites" to make that clearer?

[the remark this answers is not in the document]
Comment (oliverdunk) [reply site/en/docs/extensions/reference/cookies/index.md:29]: I don't think that's quite right - we're talking about a single site C here, just in three different contexts (at the top-level, or in an iframe on two different sites).

[the remark this answers is not in the document]
Comment (jpmedley) [reply site/en/docs/extensions/reference/cookies/index.md:29]: How about "A partitioned cookie from site A will have different values when embedded on sites B and C."

The top-level case isn't being discussed [in this context](https://developer.chrome.com/docs/extensions/reference/cookies/#partitioning).

[the remark this answers is not in the document]
Comment (oliverdunk) [reply site/en/docs/extensions/reference/cookies/index.md:29]: So the piece that Privacy Sandbox folk have called me out for in the past is that it's not a single cookie with three values - it's three different cookies that have their own values and potentially different other properties as well.

I don't feel strongly on this and I think your wording gets the point across. But just wanted to call that out.

File: site/en/docs/extensions/reference/cookies/index.md [hunk 3]
```diff
@@ -25,10 +25,10 @@ to access. For example:
...
```
Comment (jpmedley) [inline site/en/docs/extensions/reference/cookies/index.md:29]:
```suggestion
and site C, a partitioned cookie from site A will have different values when embedded on sites B and C.
```

Comment (wizardofdocs) [reply site/en/docs/extensions/reference/cookies/index.md]: how's this?

Comment (oliverdunk) [reply site/en/docs/extensions/reference/cookies/index.md]: I'm fine with that :)

**Review by jpmedley — APPROVED** [review]

**Pull request merged**